runtime/oak/src/lib.rs

```diff
@@ -1040,15 +1040,15 @@ pub struct ClosedCallFilter;
 impl Contains<Call> for ClosedCallFilter {
 	fn contains(c: &Call) -> bool {
 		match c {
-			Call::AutomationTime(_) => false,
-			Call::Balances(_) => false,
-			Call::Bounties(_) => false,
-			Call::Currencies(_) => false,
-			Call::ParachainStaking(_) => false,
-			Call::PolkadotXcm(_) => false,
-			Call::Treasury(_) => false,
-			Call::XTokens(_) => false,
-			Call::AutomationPrice(_) => false,
+			Call::AutomationTime(_) => true,
```

Comment:
why did you swap these to `true`?

Reply:
isn't this the valve section?

Reply:
It is for the valve panic button. Anything in here that returns false will be filtered when we turn valve off completely. This is separate and overrides the gates.